src/include/platform/ConnectivityManager.h

```diff
@@ -289,12 +289,15 @@ struct ConnectivityManager::SEDIntervalsConfig
 };
 
 #if CHIP_DEVICE_CONFIG_ENABLE_WIFIPAF
+#define NAN_FREQ_LIST_ALL 0xff
```

Comment:
use `constexpr uint8_t kNaNFreqListAll` (or whatever type or name is apropriate instead) to get type safety as well.

Reply:
I will fix it